package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gisatcz/ptr-tile-grid",
-  "version": "0.0.1",
+  "version": "0.1.0",
```

Comment:
Why minor? The library is not even complete yet.

Reply:
Changes are not fixes, it`s new functionality.